bundles/com.espressif.idf.serial.monitor/src/com/espressif/idf/serial/monitor/core/IDFMonitor.java

```diff
@@ -86,8 +88,12 @@ private String getMonitorBaudRate()
 		return new SDKConfigJsonReader(project).getValue("ESPTOOLPY_MONITOR_BAUD"); //$NON-NLS-1$
 	}
 
-	public Process start() throws IOException
+	public Process start() throws Exception
 	{
+		if(!dependenciesAreInstalled())
+		{
+			throw new Exception("Missing Dependencies"); //$NON-NLS-1$
```

Comment:
According to best practice, we should treat Exception as an abstract class and therefore not use it directly, because it tells us little about what we expect to go wrong. 
Maybe we can change the text to something more informative, like "The WebSocket dependency is missing and cannot be installed automatically"?

Reply:
@sigmaaa I have updated the exception message